azure-pipelines.yml

```diff
@@ -252,6 +252,9 @@ jobs:
       runCategories: 'runtime libraries'
       channels:
         - nativeaot7.0
+      variables:
```

Comment:
Applying a fix to yml file solves the problem for the CI, but not for the users who run the benchmarks locally using the python script (this includes our monthly runs).

In case you need to disable some AOT benchmarks in the future you can use `AotFilter` attribute like here:

https://github.com/dotnet/performance/blob/6e1585f375b41e1486420f264b0704d5f24d3694/src/benchmarks/micro/runtime/System.Reflection/Activator.cs#L26

This works for all scenarios.